packages/contracts/test/token/erc20/governance-erc20.ts

```diff
@@ -246,7 +249,11 @@ describe('GovernanceERC20', function () {
       // but the transfer should turn it on.
       expect(await token.delegates(signers[1].address)).to.equal(addressZero);
 
-      await token.transfer(signers[1].address, 50);
+      // Should turn on delegation
```

Comment:
despite everything being named `delegate...` this turns on the `votingPower` calculations. I would write that as others were confused by it yesterday

Reply:
Can this be a follow-up task?

Reply:
I meant the comments / test descriptions. Not worth a separate task IMO